packages/storage/README.md

```diff
@@ -53,6 +53,11 @@ localReadStream.pipe(remoteWriteStream);
 bucket.upload('/photos/zoo/zebra.jpg').then(function(data) {
   var file = data[0];
 });
+
+// It's also possible to integrate with third-party Promise libraries.
```

Comment:
Part of me thinks it's ok to leave this here since it's what gets displayed on npm for the package overview. I think if anything we can just be much more elaborate on the umbrella docs.